<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: black;
}</string>
   </property>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/powerButton.svg);
background-color: black;
border-radius: 35px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="downButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/downButton.svg);
background-color: black;
border-radius: 35px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="upButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/upButton.svg);
background-color: black;
border-radius: 35px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
	font-size: 15pt;
}</string>
    </property>
    <property name="text">
     <string>Oasis Pro</string>
    </property>
   </widget>
   <widget class="QLabel" name="canada">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	color: white;
	font-size: 15pt;
}</string>
    </property>
    <property name="text">
     <string>Canada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/okButton.svg);
background-color: black;
border-radius: 35px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/backButton.svg);
background-color: black;
border-radius: 35px;
}

QPushButton:pressed {
	background-color: #ffd300;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="intensityBar">
       <item>
        <widget class="QLabel" name="powerLevel8">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-8-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel7">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-7-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel6">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-6-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel5">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-5-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel4">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-4-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel3">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-3-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel2">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-2-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="powerLevel1">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel{
	color: black;
	font-style: bold;
	font-size: 15pt;
}</string>
         </property>
         <property name="text">
          <string>-1-</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="mainDisplay">
       <property name="styleSheet">
        <string notr="true">QListWidget{
	color:white;
	font-size: 15pt;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="LED">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>120</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QSlider" name="batterySlider">
       <property name="styleSheet">
        <string notr="true">QSlider{
	background-color: black;
}

QSlider::handle{
	background-color: black;
	border: 1px solid white;
}</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="batteryLabel">
       <property name="styleSheet">
        <string notr="true">QLabel{
	color: white;
	font-size: 15pt;
}</string>
       </property>
       <property name="text">
        <string>Battery</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QProgressBar" name="batteryLevelBar">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QCheckBox" name="connectionCheck">
       <property name="styleSheet">
        <string notr="true">QCheckBox{
	background-color: black;
	font-size: 15pt;
	color: white;
}

QCheckBox::indicator{
	border: 1px solid white;
}

QCheckBox::indicator:checked{
	background-color: white;
}</string>
       </property>
       <property name="text">
        <string>Connection Status</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QGraphicsView" name="screen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>130</y>
      <width>341</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryWarning">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>450</y>
      <width>277</width>
      <height>37</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 20pt;
	color: red;
}</string>
    </property>
    <property name="text">
     <string>LOW BATTERY</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>batterySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>batteryLevelBar</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>690</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>690</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
